Makefile

```diff
@@ -0,0 +1,19 @@
+all: push
+
+# See pod.yaml for the version currently running-- bump this ahead before rebuilding!
+TAG = 0.1
+
+# TODO(random-liu): Change the project to google_containers.
+PROJ = google.com/noogler-kubernetes
```

Comment:
I don't think it has changed. Do you need instructions on how to push to gcr.io/google-containers